pubspec.yaml

```diff
@@ -54,7 +54,7 @@ dependencies:
   html_editor_enhanced:
     git:
       url: https://github.com/linagora/html-editor-enhanced.git
-      ref: email_supported
+      ref: change-flex-color-picker-lib
```

Comment:
why updated this library?

Reply:
Should be `master`

Reply:
-`html-editor-enhanced=`This library also uses `html-editor-enhanced`. So I switched to my branch for testing before being merged into master. I updated it again

Plss check my PR: https://github.com/linagora/html-editor-enhanced/pull/9